bincompat-c-hello/scripts/common.sh

```diff
@@ -0,0 +1,17 @@
+ELFLOADER_BASE="elfloader-basic"
```

Comment:
```suggestion
#!/bin/sh

elfloader_base="elfloader-basic"
```

Capitals in variable names are usually reserved.